podder_task_foundation/objects/dictionary.py

```diff
@@ -44,11 +44,11 @@ def save(self, path: Path) -> bool:
             return False
 
         if file_type == "yaml":
-            path.write_text(yaml.dump(self._data))
+            path.write_text(yaml.dump(self._data), encoding='utf-8')
```

Comment:
Should not use fixed "utf-8" because the save method won't allow using other encodings.

So, you should add optional encoding param on the save method.
```
def save(self, path: Path, encoding: Optional[str] = 'utf-8') -> bool:
```

Reply:
And do the same modification on array.py

Reply:
Great! Thank you so much for your comment. I fixed them.